package-lock.json

```diff
@@ -1,2756 +1,8 @@
 {
   "name": "device-types",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
please upgrade node and run npm install again ;)

Reply:
I just installed both, nodejs has version 12.18.2, npm is version 6.14.8 :thinking:

Reply:
it should be v15.12.0 and 7.6.3

Reply:
well, npm is not 7.7.4 :man_shrugging: